integration/test/Testlib/App.hs

```diff
@@ -49,3 +50,12 @@ ownDomain = asks (.domain1)
 
 otherDomain :: App String
 otherDomain = asks (.domain2)
+
+-- | Run an action, `recoverAll`ing with exponential backoff (min step 8ms, total timeout
+-- ~30s).  Search this package for examples how to use it.
```

Comment:
15s max would also be ok for me.  or 10?

Reply:
Yeah, our setup _should_ be able to resolve things within 15s, if not, then maybe something else is wrong?